ModelicaTest/MultiBody.mo

```diff
@@ -441,9 +439,7 @@ for rodLength:
         n={1,0,0},
         a(fixed=false),
         s(fixed=true),
-        v(fixed=false),
-        s_offset=-0.2) annotation (Placement(transformation(extent={{12,-80},{
-                32,-60}})));
+        v(fixed=false)) annotation (Placement(transformation(extent={{12,-80},{32,-60}})));
```

Comment:
Deleting `s_offset` here results in an offset in `j2.s` by -0.2 compared to reference results, which is fine. The consequence is that the reference results have to be generated again for further tests. This holds similarly for other test models.
![Clipboard22](https://user-images.githubusercontent.com/9588978/61366150-50bc1480-a889-11e9-8844-6dc6f9ef0811.jpg)

Reply:
OK, joints are moved to yield reproducible results.